OxfordBodleian/Juel-Jensen/BDLaethd19.xml

```diff
@@ -456,26 +506,958 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                   </layout>
+                            </layoutDesc>
+                    
+                        </objectDesc>
+                    
+                        <additions>
+                            <list>
+                                <item xml:id="a5">
+                                    <locus target="#150r"/>
+                                    <desc type="Unclear">In the blank spaces of the headpiece decorating the beginning of the 
+                                        <ref type="work" corresp="LIT2509Weddas"/>, there is a barely legible note written with
+                                        red ink, most probably by <ref type="hand" corresp="#h1"/>.
+                                    </desc>
+                                    <q xml:lang="gez">
+                                        <gap reason="illegible"/><hi rend="rubric">ያም<gap reason="illegible"/> ሊተ፡ ለ<gap reason="illegible"/>
+                                            <gap reason="illegible"/>ተዝካር፡ ፍቅረ፡ ዘማእከሌ<gap reason="illegible"/>
+                                            <gap reason="illegible"/>ወሰላም፡ ለከ</hi>
+                                    </q>
+                                </item>
+                                
+                                <item xml:id="a6">
+                                    <locus from="165ra" to="165vb"></locus>
+                                    <desc type="GuestText">The beginning of <ref type="work" corresp="LIT6763IntroEgzNagsa"/>. The addition is
+                                        written in hand different from that of the main text but of approximately the same age.
+                                    </desc>
+                                    <q xml:lang="gez">
+                                        <seg part="I">
+                                            <lb/><hi rend="rubric">በስመ፡ እግዚኣብሔር፡ ስ</hi>
+                                            <lb/>ሉስ፡ ኅቡረ፡ ሀልዎ፡ ልዑ
+                                            <lb/><hi rend="rubric">ል፡ ዘኢይረክብዎ፡ መ</hi>
+                                            <lb/>ንፈስ፡ ዚኢገስስዎ፡
+                                            <add place="above">ክቡት፡ ዘኢይሬእይዎ፡ እሳት፡ ዘኢይለክፍዎ፡</add>
+                                            <lb/>ኃይል፡ <choice><orig>ዚ</orig><corr>ዘ</corr></choice>ኢይሌብው
+                                            <lb/>ዎ፡ ንጉሥ፡ ኢይሰይም
+                                            <lb/>ዎ፡ አምላክነ፡ በመለ<supplied reason="lost">ኮ</supplied>
+                                            <lb/>ቱ፡ ወአቡነ፡ በኂሩቱ፡ <add place="inline"><unclear>ዘ</unclear></add>አ
+                                            <lb/>ሐደ፡ ይሰገድ፡ <add place="inline">ወ</add>አሐደ፡ ይሴ
+                                            <lb/>ባሕ፡ <add place="inline">ወ</add>አሐደ፡ ይጼለይ፡ ሎ
+                                            <lb/>ቱ፡ ስብሐት፡ ለዓለመ፡ ዓ
+                                            <lb/>ለም፡ አሜን<hi rend="rubric" rendition="#partialRubric">፨=፨</hi>
```

Comment:
I think we opted for reducing this punctuation mark ፨=፨ to simple ፨ in our descriptions, but I might be wrong. @thea-m

Reply:
I'm afraid I don't remember and cannot find any notes on this in our documentation...

Reply:
It seems like the string ' ፨=፨ ' has not been used elsewhere in J-J, so I'll take it away and assume that you have been standardizing :)